$NetBSD: patch-ap,v 1.1 2006/04/09 06:34:32 jlam Exp $

On VAX, feeding a bare "NaN" to nawk causes nawk to core dump since
it tries to interpret it as a number, which causes an FP exception.

--- MANIFEST.orig	2006-01-31 18:27:53.000000000 -0500
+++ MANIFEST
@@ -1586,7 +1586,7 @@ lib/Math/BigInt/t/const_mbf.t	Test Math:
 lib/Math/BigInt/t/downgrade.t	Test if use Math::BigInt(); under downgrade works
 lib/Math/BigInt/t/_e_math.t	Helper routine in BigFloat for _e math
 lib/Math/BigInt/t/fallback.t	Test Math::BigInt
-lib/Math/BigInt/t/inf_nan.t	Special tests for inf and NaN handling
+lib/Math/BigInt/t/inf_nan.t	Special tests for inf and *NaN* handling
 lib/Math/BigInt/t/isa.t		Test for Math::BigInt inheritance
 lib/Math/BigInt/t/lib_load.t	Test sane lib names
 lib/Math/BigInt/t/mbf_ali.t	Tests for BigFloat
